src/backend/cgcod.c

```diff
@@ -726,7 +726,7 @@ code *prolog()
     assert(usedalloca != 1);
     AllocaOff = alignsection(usedalloca ? (Foff - REGSIZE) : Foff, REGSIZE, bias);
 
-    CSoff = alignsection(AllocaOff - cstop * REGSIZE, REGSIZE, bias);
+    CSoff = alignsection(AllocaOff - (int)cstop * REGSIZE, REGSIZE, bias);
```

Comment:
Doing a grep for cstop:
```
cgcod.c:unsigned cstop;                 /* # of entries in CSE table (csextab[])   */
cgcod.c:    cstop = 0;                          /* no entries in table yet      */
cgcod.c:    while (cstop != 0 && (csextab[cstop - 1].flags & CSEload) == 0)
cgcod.c:        cstop--;
cgcod.c:    CSoff = alignsection(AllocaOff - cstop * REGSIZE, REGSIZE, bias);
cgcod.c:        for (unsigned i = 0; i < cstop; i++)
cgcod.c:    for (unsigned i = cstop; ms; i++)
cgcod.c:        if (i >= cstop)
cgcod.c:            memset(&csextab[cstop],0,sizeof(csextab[0]));
cgcod.c:        if (csextab[i].e == NULL || i >= cstop)
cgcod.c:            if (i >= cstop)
cgcod.c:                cstop = i + 1;
cgcod.c:  for (unsigned i = 0; i < cstop; i++)
cgcod.c:        for (unsigned i = cstop; i--;)           /* look through saved comsubs   */
cgcod.c:  for (unsigned i = cstop; i--;)
```
I don't see any way `cstop` can go negative unless there are (2^32)/8 common subexpressions, which isn't going to happen. So I don't think this fixes anything.